apps/potlock/widget/utils.jsx

```diff
@@ -234,7 +234,7 @@ return {
         _num += 1;
         _sum = _sum.plus(v1);
         for (const [k2, v2] of Object.entries(contribz)) {
-          if (k2 > k1) {
+          if (k2 > k1 || Object.keys(contribz).length === 1) {
```

Comment:
@M-Rb3 for reference can you explain the problem that required this change in the quadratic calculation code?

Reply:
![image](https://github.com/PotLock/bos-app/assets/68287884/6ab25de6-a1f0-4c91-87a6-9b1f66a64df1)

![image](https://github.com/PotLock/bos-app/assets/68287884/66b94557-9296-49c4-84a0-f019cc8cc05b)

it should be unnecessary to add it because bigtot.gte(totalPot). idk why it was not working, I ended up updating the condition and I saw everything worked fine and left it as it